xml/src/test/java/org/ehcache/xml/service/DefaultCacheLoaderWriterConfigurationParserTest.java

```diff
@@ -57,4 +58,12 @@ public void unparseServiceConfiguration() {
 
     assertThat(cacheType.getLoaderWriter().getClazz()).isEqualTo(TestCacheLoaderWriter.class.getName());
   }
+
+  @Test(expected = XmlConfigurationException.class)
```

Comment:
I really don't like expected exceptions. Because you never know if the exception was thrown from the right place. And I like to check the message as well. I use assertJ instead: http://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#exception-assertion